pytest/tests/mocknet/mirror.py

```diff
@@ -437,6 +437,14 @@ def run_remote_cmd(args, traffic_generator, nodes):
          on_exception="")
 
 
+def run_env_cmd(args, traffic_generator, nodes):
+    if args.clear_all:
```

Comment:
So if I do `env --clear-all --key-value "FOO=bar"`, the `--key-value` arg is just silently ignored, which seems pretty unexpected. Not to harp too much on it, but I think it would really make sense to do what I suggested in the other comment, and make the `--clear-all` part just add an extra arg to the RPC query, and then that would not only make things better with only one RPC round trip, but it would also fix this UX issue